proto/internalapi/sensor/collector_iservice.proto

```diff
@@ -3,16 +3,22 @@ syntax = "proto3";
 package sensor;
 
 import "google/protobuf/empty.proto";
-import "internalapi/common/collector_runtime_config.proto";
+import "storage/cluster.proto";
 
 option go_package = "./internalapi/sensor;sensor";
 
 message MsgToCollector {
   oneof msg {
-    common.CollectorRuntimeConfig config_with_cluster = 1;
+    storage.CollectorConfig collector_config = 1;
```

Comment:
We usually reuse the same number when we are fully safe regarding backwards compatibility. Was there a release where `common.CollectorRuntimeConfig config_with_cluster = 1;` was included? 
If yes, we need to mark `1` as reserved and give the `collector_config` id `2`.

Reply:
This file does not exist in the release-4.5 branch.